### PR TITLE
Test semgrep

### PR DESCRIPTION
**Pull request by bcaller**

### CODE REVIEW

**Review by thypon — APPROVED** [review]

File: t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+async function x() {
+    // ruleid: path-travesal-by-string-interpolation
+    await fetch(`${env.API_HOSTNAME}/api/subscriptions/${subscriptionId}`, 1337);
```
[the remark this answers is not in the document]
Comment (thypon) [reply t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts:3]: Why none of these findings contains the `Cc` ?

[the remark this answers is not in the document]
Comment (thypon) [reply t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts:3]: ^^ @bcaller

[the remark this answers is not in the document]
Comment (bcaller) [reply t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts:3]: Interesting

[the remark this answers is not in the document]
Comment (bcaller) [reply t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts:3]: I missed a `\` at the end of the sed line in bash

[the remark this answers is not in the document]
Comment (bcaller) [reply t3sts/semgrep_rules/path-travesal-by-string-interpolation.server.ts:3]: Why does that not cause an error?

**Review by thypon — APPROVED** [review]

**Pull request merged**